core/modules/field_ui/js/field_ui.js

```diff
@@ -93,7 +94,10 @@ jQuery.fn.fieldUIPopulateOptions = function (options, selected) {
     jQuery.each(options, function (value, text) {
       // Figure out which value should be selected. The 'selected' param
       // takes precedence.
-      var is_selected = ((typeof selected != 'undefined' && value == selected) || (typeof selected == 'undefined' && text == previousSelectedText));
+      var is_selected = (
+        (typeof selected != 'undefined' && value == selected) ||
+        (typeof selected == 'undefined' && text == previousSelectedText) ||
+        (value == widgetDefault));
```

Comment:
I think this last condition needs to check that `selected` is undefined. Otherwise if a default value is set and it is different from the widget default, we may end up with two `<option selected="selected">` tags.

So I think the last line should be:
```
(typeof selected == 'undefined' && value == widgetDefault));
```